skeletons/web-extension/scripts/in-page-script.js

```diff
@@ -29,17 +29,9 @@
   onReady(function() {
     var libraries = window.Ember && window.Ember.libraries;
     if (libraries) {
-      // Ember has changed where the array of libraries is located.
-      // In older versions, `Ember.libraries` was the array itself,
-      // but now it's found under _registry.
-      if (libraries._registry) {
-        libraries = libraries._registry;
-      }
-
-      var versions = Array.prototype.slice.call(libraries, 0);
       window.postMessage({
         type: 'emberVersion',
-        versions: versions
+        versions: libraries._registry.slice()
```

Comment:
This script runs for all Ember versions (even < 2.7), so let's keep the backwards compatibility here.